src/HealthMonitor/Checks/HealthCheck.php

```diff
@@ -9,4 +9,6 @@ interface HealthCheck
     public function check(): bool;
 
     public function message(): string;
+
+    public function notifiers(): array;
```

Comment:
hier kan je misschien ook een docblock met 
@return Notifier[] zodat we weten dat elk element van de array een Notifier instance moet zijn